docs/tasks/issuers/setup-vault.rst

```diff
@@ -207,3 +207,57 @@ For more information on ClusterIssuers, read the
 :doc:`ClusterIssuer reference docs </reference/clusterissuers>`.
 
 .. _`Subject Alternative Names`: https://en.wikipedia.org/wiki/Subject_Alternative_Name
+
+Vault Authentication with Kubernetes Service Accounts
+=====================================================
+
+This Vault authentication method uses Service Account tokens created by
+Kubernetes to authenticate requests to Vault for signing certificates. You can
+find more information on how to configure vault for Kubernetes based Service
+Account authentication in the `documentation
+<https://www.vaultproject.io/docs/auth/kubernetes.html>`__. This authentication
```

Comment:
Is two underscores valid? 🤔

Reply:
Thought was the way it was supposed to be, it's used elsewhere. Is that incorrect?